code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -170,7 +170,7 @@
 	new /obj/item/reagent_containers/food/pill/patch/synthflesh(src)
 	new /obj/item/reagent_containers/food/pill/patch/synthflesh(src)
 	new /obj/item/reagent_containers/food/pill/patch/synthflesh(src)
-	new /obj/item/defibrillator/compact/combat/loaded(src)

```

Comment:
This is kinda the biggest reason you buy the kit. I think people severely underestimate the strength of this thing--it's insane, yet no one touches it.

Reply:
People shouldn't be buying medical supplies because they come with an instakill weapon

Reply:
You so rarely see medics on nuke ops. Sometimes people will occasionally purchase the medbeam gun. I feel like the combat defib is relatively unknown and inflates the price of the medical supplies box substantially. I have never seen this medical kit be purchased and the NV medhud is pretty critical for a nukie medic. I think the 7 TC price tag puts people off, you could get an emag for that or nearly an esword.

I could potentially move the combat defib into a more specialised medic bundle with stronger items to reflect a greater cost, or just have it as a standalone item as the medgun is.

Reply:
You could have two medkits, a 4TC personal basic syndi kit, and the 7TC advanced one.